project-management/sprint-reports/sprint-3.md

```diff
@@ -0,0 +1,62 @@
+* [`#170`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/170): Prototype: Develop layout implementation of the landing page Hero/Jumbotron component 
+* [`#171`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/171): Prototype: Develop hamburger menu component
+* [`#172`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/172): Prototype: Develop landing page footer
+* [`#176`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/171): Document the process of how we will get feedback from the community
+* [`#179`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/179): Clarify how people can give feedback on the sprint demo video
+* [`#188`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/188): Create a community involvement strategy 
+* [`#204`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/204): Gather feedback on videos each sprint
+
+### Issues not progressed
+
+* [`#166`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/166): Translatable platform
+* [`#169`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/169): Prototype: Develop layout of landing page function boxes
+* [`#173`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/174): Prototype: Develop paired down secured engagement component
+* [`#174`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/174): Prototype: Develop "add your experience" page component
+
+### Issues opened
+
+These tasks were opened during the sprint but no work was done to address them.
+
+* [`#200`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/200): Include non-verbal means of inputting information
+* [`#200`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/200): Include non-verbal means of inputting information
+* [`#229`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/229): Prototype: Develop template editor component
+* [`#230`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/200): Prototype: Develop newsletter subscription control
+* [`#231`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/231): Prototype: Develop stepped user journey control
+* [`#232`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/232): Prototype: Develop language selector control
+* [`#233`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/233): Prototype: Develop user profile type selection control information
+ 
+
+### Challenges
+
+
+No challenges were highlighted in this sprint.
+
+### Request for community feedback
+
+Feedback is requested from the community on the following tasks.
+
+* [#182](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/182): issue to collect feedback on sprints 1 & 2
```

Comment:
Does the name on this issue need to be changed? It says to collect feedback from sprints 1 & 2?